examples/tracing/bitehist.py

```diff
@@ -16,22 +16,31 @@
 from __future__ import print_function
 from bcc import BPF
 from time import sleep
+import os
+
+# check "blk_account_io_completion" against /proc/kallsyms
+string = os.popen('cat /proc/kallsyms | ' + 
+			'grep "blk_account_io_completion"'
+			).read()
+if len(string.strip()) > 0: 
+    event = "blk_account_io_completion"
+else: 
+    event = "blk_account_io_done"
```

Comment:
since you mentioned biolatency just switched to io_done straight out, and it exists in older versions, we can just do a simple swap